### PR TITLE
link to SSP and Project Plan templates

### LINKED ISSUE #289: updated ssp low template
template for low systems leveraging cloud.gov
https://docs.google.com/a/gsa.gov/document/d/1tVbH39TFfvSaBbjWfLaR3GLOuvsLuhLFJ75xKowEV5c/edit?usp=sharing

### PR DESCRIPTION
**Pull request by afeld**
Closes #289.

### CODE REVIEW

File: _includes/checklist.md [hunk 1]
```diff
@@ -24,7 +24,9 @@ Everything in this section needs to be completed before the project will be sche
     - [ ] Add Rules of Engagement (RoE) template
         * Search [this page](https://insite.gsa.gov/portal/content/627238) for "Rules of Engagement (RoE) 90-Day LATO Penetration Test TEMPLATE", even if this isn't for a 90-day LATO.
     - [ ] Add [System Security Plan (SSP)](https://pages.18f.gov/before-you-ship/ato/ssp/) template
+        * For Low systems on cloud.gov, use [this template](https://docs.google.com/a/gsa.gov/document/d/1tVbH39TFfvSaBbjWfLaR3GLOuvsLuhLFJ75xKowEV5c/edit?usp=sharing)
```
Comment (afeld) [inline _includes/checklist.md]: Still waiting on access to this, so let's wait to merge until I can confirm this link is right.

File: _includes/checklist.md [hunk 2]
```diff
@@ -24,7 +24,9 @@ Everything in this section needs to be completed before the project will be sche
...
     - [ ] Add Project Plan template
+        * Search [this page](https://insite.gsa.gov/portal/content/627238) for "One Year LATO Project Plan Template", even if this isn't for a one-year LATO.
```
Comment (afeld) [inline _includes/checklist.md]: Is this correct?

Comment (hollyallen) [inline _includes/checklist.md]: Yes, right now this is correct.

File: _includes/checklist.md (hunk 1 above)
Comment (hollyallen) [inline _includes/checklist.md]: I can verify this link is correct. I've sent another document access request to Jay.

**Comment by hollyallen** [comment]
this looks good to go, your preference wrt waiting on the doc access.

File: _includes/checklist.md (hunk 1 above)
Comment (afeld) [inline _includes/checklist.md]: Looks like we both got access, which is good enough for now.

**Pull request merged**